spec/unit/settings/server_list_setting_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+require 'puppet/settings/server_list_setting'
+
+describe Puppet::Settings::ServerListSetting do
+
+    it "prints strings as strings" do
+      settings = Puppet::Settings.new
+      settings.define_settings(:main, neptune: {type: :server_list, desc: 'list of servers'})
+      server_list_setting = settings.setting(:neptune)
+      expect(server_list_setting.print("jupiter,mars")).to eq("jupiter,mars")
```

Comment:
Question: Since ServerListSetting is-a ArraySetting, why would the print method ever recieve a string?

Reply:
The raw data from the puppet.conf or the input on command line before it has been transformed by Puppet into an array could be a string of whatever the user has entered.

Reply:
To add on to @jtappa's comment: This especially happens when the print method is hit from `puppet config print`.